pages/settings/SettingsNode.qml

```diff
@@ -362,6 +362,7 @@ Rectangle{
                     if (appWindow.daemonRunning) {
                         appWindow.stopDaemon();
                     } else {
+                        persistentSettings.daemonFlags = daemonFlags.text;
```

Comment:
Is this required? `onEditingFinished: persistentSettings.daemonFlags = daemonFlags.text;` should do this anyway. I don’t know if the signal gets emitted before the daemon starts though.

Reply:
@mmbyday makes sense :) thanks

Reply:
@selsta Unfortunately, signal `onEditingFinished` is only emitted when user leaves the input. In the situation that the user is still focused (editing) the input box and clicks `Start Daemon`, the event is not emitted :(